modules/security-jwt/src/main/resources/application.properties

```diff
@@ -0,0 +1,3 @@
+#security.authentication.keystore.keystorelocation=C:/Devon-dist_3.0.0/jwtkeystore.p12
```

Comment:
we should make explicit that these are only some examples. Further, it only makes sense to add reasonable defaults here. Some more reasonable default could be `~/keystore.p12` or `~/jwt-keystore.p12` as long as we would support resolving `~` to `user.home` system property value (do not know if Java NIO already does that or we still would have to code that explicitly).